File: src/main/java/ru/ioleksiv/telegram/bot/core/api/model/objects/inline/queryresult/InlineQueryResultCachedAudio.java
package ru.ioleksiv.telegram.bot.core.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @see <a href="https://core.telegram.org/bots/api#inlinequeryresultcachedaudio>InlineQueryResultCachedAudio</a>
 */
public class InlineQueryResultCachedAudio extends FormattedInlineResult {
    private static final String TYPE = "audio";
    /**
     * audio_file_id	String	A valid file identifier for the audio file
     */
    @JsonProperty("audio_file_id")
    private String audioFileId = null;

    InlineQueryResultCachedAudio() {
        super(TYPE);
    }

    public String getAudioFileId() {
        return audioFileId;
    }

    public void setAudioFileId(String audioFileId) {
        this.audioFileId = audioFileId;
    }

}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultCachedAudio.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

/**
 *  [InlineQueryResultCachedAudio](https://core.telegram.org/bots/api/#inlinequeryresultcachedaudio)
 */
class InlineQueryResultCachedAudio(
        /**
         * audio_file_id String A valid file identifier for the audio file
         */
        @JsonProperty("audio_file_id")
        var audioFileId: String? = null,
        /**
         * type String Type of the result
         */
        @JsonProperty("type")
        val type: String = "audio"
) : FormattedInlineResult()
